apps/dokploy/components/dashboard/compose/monitoring/show.tsx

```diff
@@ -1,86 +1,115 @@
 
-	return (
-		<div>
-			<Card className="bg-background">
-				<CardHeader>
-					<CardTitle className="text-xl">Monitoring</CardTitle>
-					<CardDescription>Watch the usage of your compose</CardDescription>
-				</CardHeader>
-				<CardContent className="flex flex-col gap-4">
-					<Label>Select a container to watch the monitoring</Label>
-					<Select onValueChange={setContainerAppName} value={containerAppName}>
-						<SelectTrigger>
-							<SelectValue placeholder="Select a container" />
-						</SelectTrigger>
-						<SelectContent>
-							<SelectGroup>
-								{data?.map((container) => (
-									<SelectItem
-										key={container.containerId}
-										value={container.name}
-									>
-										{container.name} ({container.containerId}) {container.state}
-									</SelectItem>
-								))}
-								<SelectLabel>Containers ({data?.length})</SelectLabel>
-							</SelectGroup>
-						</SelectContent>
-					</Select>
-					<DockerMonitoring
-						appName={containerAppName || ""}
-						appType={appType}
-					/>
-				</CardContent>
-			</Card>
-		</div>
-	);
+  const { mutateAsync: restart } = api.docker.restartContainer.useMutation();
+
+  useEffect(() => {
+    if (data && data?.length > 0) {
+      setContainerAppName(data[0]?.name);
+      setContainerId(data[0]?.containerId);
+    }
+  }, [data]);
+
+  return (
+    <div>
+      <Card className="bg-background">
+        <CardHeader>
+          <CardTitle className="text-xl">Monitoring</CardTitle>
+          <CardDescription>Watch the usage of your compose</CardDescription>
+        </CardHeader>
+        <CardContent className="flex flex-col gap-4">
+          <Label>Select a container to watch the monitoring</Label>
+          <div className="flex flex-row gap-4">
+            <Select
+              onValueChange={(value) => {
+                setContainerAppName(value);
+                setContainerId(
+                  data?.find((container) => container.name === value)
+                    ?.containerId,
+                );
+              }}
+              value={containerAppName}
+            >
+              <SelectTrigger>
+                <SelectValue placeholder="Select a container" />
+              </SelectTrigger>
+              <SelectContent>
+                <SelectGroup>
+                  {data?.map((container) => (
+                    <SelectItem
+                      key={container.containerId}
+                      value={container.name}
+                    >
+                      {container.name} ({container.containerId}){" "}
+                      {container.state}
+                    </SelectItem>
+                  ))}
+                  <SelectLabel>Containers ({data?.length})</SelectLabel>
+                </SelectGroup>
+              </SelectContent>
+            </Select>
+            <Button
+              onClick={async () => {
+                toast.success(`Restarting container ${containerId}`);
```

Comment:
instead of containerId, can we show the container name?

Reply:
yep, it will be better